services/rpcfilters/block_provider.go

```diff
@@ -55,3 +55,7 @@ func (p *latestBlockProviderRPC) GetLatestBlock() (blockInfo, error) {
 
 	return result, nil
 }
+
+func (p *blockProviderRPC) GetBlocksInRange(start int64, end int64) ([]blockInfo, error) {
```

Comment:
Here I can call `eth_getBlockByNumber` over and over to get all the blocks I need. How do go from list of blocks to list of logs? I will be left with blocks in this format: https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getblockbyhash

Reply:
[`eth_getLogs`](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getlogs) already gets *from* and *to* and other parameters, I think it might be wise to use it instead.